src/service_account.rs

```diff
@@ -25,10 +25,13 @@ use types::{StringError, Token};
 use hyper::header;
 use url::form_urlencoded;
 
+use rustls::{self, PrivateKey};
+use rustls::sign::{self, Signer};
+use rustls::internal::pemfile;
```

Comment:
Not too sure but hyper-rustls seems to use it happily in its examples.
https://github.com/ctz/hyper-rustls/blob/master/examples/server.rs#L15